### DISCUSSION TITLE
Official Buds Pro UWP app for Win10 has been released in MS Store

### DISCUSSION THREAD

**Discussion by cozyplanes** [timschneeb/GalaxyBudsClient#224] [Show and tell]
https://www.microsoft.com/en-us/p/galaxy-buds/9nhtlwtkfznb?activetab=pivot:overviewtab

**Comment by cozyplanes** [comment]
Galaxy Buds Live and Buds+ is scheduled to be supported later. Pro is only supported as of now. 

![](https://i.imgur.com/O0s6XHY.png)![](https://i.imgur.com/eWC2GTp.jpg)![](https://i.imgur.com/71UBgHx.jpg)![](https://i.imgur.com/crwc7zi.jpg)

**Comment by timschneeb** [comment]
Oh wow, I never expected them to release an actual Windows app.

I'll just hope that they won't DMCA this project to force this app out of existence now that they have their own app.

**Comment by timschneeb** [comment]
This is a UWP app, so it doesn't seem like we can expect macOS compatibility soon (unless they develop a completely separate app).

**Comment by stylemessiah** [comment]
I dont use an UWP apps or even use Microsofts App Store, because my desktop
isnt a smartphone,
so ill stick to using GalaxyBudsClient

Its also a great way to program a set of Buds for my poor Apple cult member
friends, who despite being
told their phone supports Buds, find out Apple doesnt...i connect their
Buds up to my PC and program the
settings via GalaxyBudsClient for them


On Thu, 13 May 2021 at 20:29, Tim Schneeberger ***@***.***>
wrote:

> This is a UWP app, so it doesn't seem like we can expect macOS
> compatibility soon (unless they develop a completely separate app).
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ThePBone/GalaxyBudsClient/issues/219#issuecomment-840469332>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACO4RXOUMWU34B5EM263Z6LTNOSZNANCNFSM44Z7Z73A>
> .
>


-- 
I hate to advocate *drugs*, *alcohol*,* violence *or
*insanity* to anyone, *but* they've *always* worked for* me*

- Hunter S. Thompson

**Comment by cozyplanes** [comment]
I don't know if it is intended or not but I really love the touchpad customization options in your client.
I set the left side for ANC on/off and right side for volume up, which is not possible when using the official client.
If it is a bug, please don't fix it 😉

**Comment by timschneeb** [comment]
> If it is a bug, please don't fix it 😉

Yeah, that was intended. 👍

Also, just to clarify, I won't stop developing this app since it still supports many things that Samsung will never implement in their apps. 
I'm also currently working on a Mac OSX port which requires me to learn Objective-C from scratch (for native implementations like a low-level Bluetooth serial client), so don't worry if overall development slows down for a while.

**Reply by sebastian-greco** [reply]
> > If it is a bug, please don't fix it 😉
> 
> Yeah, that was intended. 👍
> 
> Also, just to clarify, I won't stop developing this app since it still supports many things that Samsung will never implement in their apps. I'm also currently working on a Mac OSX port which requires me to learn Objective-C from scratch (for native implementations like a low-level Bluetooth serial client), so don't worry if overall development slows down for a while.

Is the Mac OSX port project still a thing? How can I support that development? Would be soo good to have it. Thanks for your work so far!

**Comment by ghost** [comment]
> > If it is a bug, please don't fix it 😉
> 
> Yeah, that was intended. 👍
> 
> Also, just to clarify, I won't stop developing this app since it still supports many things that Samsung will never implement in their apps.
> I'm also currently working on a Mac OSX port which requires me to learn Objective-C from scratch (for native implementations like a low-level Bluetooth serial client), so don't worry if overall development slows down for a while.

**I though the official uwp was rebuild by this repository when i suddenly searched the official uwp from Microsoft Store......Anyway thanks for sharing such a wonderful software 😍😋**

**Comment by sourajitk** [comment]
> > If it is a bug, please don't fix it 😉
> 
> Yeah, that was intended. 👍
> 
> Also, just to clarify, I won't stop developing this app since it still supports many things that Samsung will never implement in their apps.
> I'm also currently working on a Mac OSX port which requires me to learn Objective-C from scratch (for native implementations like a low-level Bluetooth serial client), so don't worry if overall development slows down for a while.

That’s great to know 👍🏼
